app/services/decisions/income_decision_tree.rb

```diff
@@ -46,6 +43,8 @@ def after_employment_status
       if not_working?
         if ended_employment_within_three_months?
           edit(:lost_job_in_custody)
+        elsif appeal_no_changes?
```

Comment:
Should this change be part of discrete change set?

Reply:
It can be if you want. I see all of these things as logic gates to which journey the user is taken through the means flow. Is there a reason you think it best to raise as a separate PR?